src/View/Shortcodes/EmbedShortcodeProvider.php

```diff
@@ -89,7 +89,24 @@ public static function handle_shortcode($arguments, $content, $parser, $shortcod
         }
 
         // Process embed
-        $embed = $embed->getEmbed();
+        try {
+            $embed = $embed->getEmbed();
+        }
```

Comment:
travis is failing because:
94 | ERROR | [x] Expected 1 space after closing brace; newline found

Reply:
think the style is supposed to be:

```suggestion
       try {
            $embed = $embed->getEmbed();
        } catch(\Embed\Exceptions\InvalidUrlException $e) {
```

https://www.php-fig.org/psr/psr-2/#56-try-catch